lib/matplotlib/colors.py

```diff
@@ -201,10 +199,11 @@ def _to_rgba_no_colorcycle(c, alpha=None):
             except KeyError:
                 pass
             else:
-                cbook.warn_deprecated(
-                    "3.1", message="Support for case-insensitive colors is "
-                    "deprecated since Matplotlib %(since)s and will be "
-                    "removed %(removal)s.")
+                if len(c) == 1:
+                    cbook.warn_deprecated(
+                        "3.1", message="Support for case-insensitive "
```

Comment:
```suggestion
                        "3.1", message="Support for uppercase "
```
'+ Please use lowercase instead.

Reply:
done